### PR TITLE
l10n: Add dot at the end of an sentence

### PR DESCRIPTION
**Pull request by rakekniven**
Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

**Review by Valdnet — APPROVED** [review]

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -198,8 +198,8 @@ export default {
 			}
 
 			return this.missingAppUpdates.length === 0
-				? t('updatenotification', '<strong>All</strong> apps have a compatible version for this Nextcloud version available', this)
-				: n('updatenotification', '<strong>%n</strong> app has no compatible version for this Nextcloud version available', '<strong>%n</strong> apps have no compatible version for this Nextcloud version available', this.missingAppUpdates.length)
+				? t('updatenotification', '<strong>All</strong> apps have a compatible version for this Nextcloud version available.', this)
+				: n('updatenotification', '<strong>%n</strong> app has no compatible version for this Nextcloud version available.', '<strong>%n</strong> apps have no compatible version for this Nextcloud version available', this.missingAppUpdates.length)
```
Comment (nickvergessen) [inline apps/updatenotification/src/components/UpdateNotification.vue:202]:
```suggestion
				: n('updatenotification', '<strong>%n</strong> app has no compatible version for this Nextcloud version available.', '<strong>%n</strong> apps have no compatible version for this Nextcloud version available.', this.missingAppUpdates.length)
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
@rakekniven please remember to use labels :)

**Comment by nickvergessen** [comment]
> Conflicting files
apps/updatenotification/js/updatenotification.js
apps/updatenotification/js/updatenotification.js.map 

Oh come on bot

**Comment by skjnldsv** [comment]
> > Conflicting files
> > apps/updatenotification/js/updatenotification.js
> > apps/updatenotification/js/updatenotification.js.map
> 
> Oh come on bot

classic! Always rebase before compiling :wink:

**Comment by rakekniven** [comment]
Could somone please resolve this conflict of packed js files and merge? Thanks.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**